packages/create-sitecore-jss/src/templates/nextjs/src/components/SafeHydrate.tsx

```diff
@@ -0,0 +1,9 @@
+﻿import dynamic from 'next/dynamic';
+import React from 'react';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const NonSSRWrapper = (props: any) => <React.Fragment>{props.children}</React.Fragment>;
+
+export default dynamic(() => Promise.resolve(NonSSRWrapper), {
```

Comment:
Wouldn't this solution completely prevent any SSR from occurring? We wouldn't want this under normal circumstances. If this is a solution we consider, it would at least need to only happen if in an editing context.

Reply:
Yes that makes sense.